### ISSUE TITLE
npm install failure

### ISSUE THREAD

**Issue by joepaquette** [tangdrew/fhir-ts#6]
I cloned your repo and attempted an `npm install` that failed:
`
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@commitlint%2fensure - User not found
npm ERR! 404
npm ERR! 404  '@commitlint/ensure@^7.5.2' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because
npm ERR! 404  1. name can only contain URL-friendly characters
npm ERR! 404 It was specified as a dependency of '@commitlint/rules'
`

Tried `npm view @commitlint/ensure@^7.5.2` and got the following:
`
npm ERR! 404 '@commitlint/ensure' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 (or use the name yourself!)
npm ERR! 404
npm ERR! 404 ...
npm ERR! 404
npm ERR! 404  '@commitlint/ensure@latest' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because
npm ERR! 404  1. name can only contain URL-friendly characters
`

Tried upgrading to the latest versions: "@commitlint/cli" to "^7.5.2" and "@commitlint/config-conventional" to "^7.5.0". Got exactly the same error as above on `npm install`.

Tried `npm cache clean -f`. Got exactly the same error as above `npm install`.

Any ideas on this dependency?

**Comment by tangdrew** [comment]
Are you using a private npm proxy? I found this [issue](https://github.com/storybooks/storybook/issues/15) after doing a little digging that seems to indicate that some old npm proxies don't support scoped npm organizations. It looks like you can do something like `npm config set @commitlint:registry https://registry.npmjs.org/` to bypass your proxy cache to fetch directly from the npm registry. Let me know if that helps!

**Comment by joepaquette** [comment]
As I understand it, we are not using a private npm proxy. I added `@commitlint:registry=https://registry.npmjs.org/` to my global `.npmrc` as you recommended, and I cleaned the cache again. That did not solve the problem. I reverted the change to my global `.npmrc`, re-cleaned the cache, and then created a project-specific `.npmrc` in the project root with only `@commitlint:registry=https://registry.npmjs.org/` in the file. That still didn't solve the problem. After reading through the issue link you provided, I discovered [this](https://github.com/storybooks/storybook/issues/15#issuecomment-251970288). As recommended, I added the _authToken and then the `npm install` succeeded.

So, here are the contents of my project-specific `,npmrc` that resolve the issue:
`
@commitlint:registry=https://registry.npmjs.org/
//registry.npmjs.org/:_authToken=00000000-0000-0000-0000-000000000000
`

Thanks for your assistance.
